### ISSUE TITLE
How to get the loss for each sample ?

### ISSUE THREAD

**Issue by familyld** [keras-team/keras#12256]
I want to write a custom loss function that weights each sample differently according the loss w.r.t each of them in the last epoch. How can I get the loss for each sample? I have tried the official example:

class LossHistory(keras.callbacks.Callback):
    def on_train_begin(self, logs={}):
        self.losses = []

    def on_batch_end(self, batch, logs={}):
        self.losses.append(logs.get('loss'))

, but logs.get('loss') only returns an average loss of that batch. How can I get the loss **for each sample**?

Please make sure that the boxes below are checked before you submit your issue.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.

Thank you!

- [ √] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [ √] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ×] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by sailormoon2016** [comment]
I have same question. anyone knows?

**Comment by sailormoon2016** [comment]
I have same question, is it possible at all?
